server/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es2022",
-    "lib": ["es2020"],
```

Comment:
This was inconsistent with the other tsconfig files and the reference config we used for Node 20:

https://www.npmjs.com/package/@tsconfig/node20.